src/test/java/baza/trainee/controller/EventControllerTest.java

```diff
@@ -0,0 +1,82 @@
+package baza.trainee.controller;
+
+
+import baza.trainee.domain.dto.event.EventDto;
+import baza.trainee.domain.dto.event.EventPreviewDto;
+import baza.trainee.service.EventService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
+
+public class EventControllerTest {
```

Comment:
use @WebMvcTest like in EventAdminControllerTest

Reply:
fixed